scripts/build.mjs

```diff
@@ -70,7 +80,20 @@ class Builder {
     // environment variable `__COMPAT_LAYER=RunAsInvoker` to force the installer
     // to run as the existing user.
     const env = { ...process.env, __COMPAT_LAYER: 'RunAsInvoker' };
+    const releaseBuildIndex = args.indexOf('--release');
+    const fullBuildVersion = this.getDescribedTag();
+    let versionPattern;
+
+    if (releaseBuildIndex !== -1) {
+      // --release => strip the commit info from the version
+      args.splice(releaseBuildIndex, 1);
+      versionPattern = /^v?(.+?)(?:-\d+-\w+)?$/;
```

Comment:
This means that production builds show `Version: 1.2.3`, where as `npm run dev` show `Version: v1.2.3`; I guess that's okay (as a subtle display of when we're running a built artifact)?

Reply:
The code-signing procedure didn't like the leading "v".

And it seemed fine to put the string exactly as returned by `git describe` in the version field for dev builds, leading 'v' and all.
The release version field is already getting edited, so dropping the leading 'v' seemed ok.